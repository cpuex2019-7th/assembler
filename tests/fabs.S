    li a1, -1063256064
    fmvwx f1, a1
    fsgnjxã€€f1, f1, f1
    fmvxw a0, f1
    
