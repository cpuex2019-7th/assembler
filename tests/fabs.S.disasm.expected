lui a1, 788992
addi a1, a1, 0
fmvwx ft1, a1
fsgnjx ft1, ft1, ft1
fmvxw a0, ft1
